### ISSUE TITLE
Maximum name length seems to be one character too long

### ISSUE THREAD

**Issue by ticky** [sonik-br/GDMENUCardManager#13]
I've set up a card using GDMenuCardManager, and once I put it on the GDEMU, the names are cut off for some of the things on it:

- "Neon Genesis Evangelion Typing Project E" is cut off to "Neon Genesis Evangelion Typing Project "
- "Official Sega Dreamcast Magazine 1999/11" is cut off to "Official Sega Dreamcast Magazine 1999/1"

I'm guessing this means that the maximum length of 40 characters is, in GDMenu's case, inclusive of the null terminator byte, because it seems like C#'s Substring method (used [here](https://github.com/sonik-br/GDMENUCardManager/blob/63e132ee70e2caab157ab1116583afa12313bd2f/src/GDMENUCardManager.Core/GdItem.cs#L35)) does what you'd expect.

**Comment by sonik-br** [comment]
I will take a look. But might take some time to test as I don't have access to my DC right now.
The app limits the char count and during my testings it was fine. But I might have missed this detail.
I don't know if the char textures in gdmenu have fixed width. If not, then this might be the problem.

**Comment by ticky** [comment]
Yep, both of these names were adapted to fit within the program's 40-character limit, but on the Dreamcast it's cut off.

Is there documentation somewhere of the GDMENU list format, or is it all based on what you've observed?

**Comment by sonik-br** [comment]
By observation. Trial and error.
There's no documentation on GDMENU.
